devicemodel/hw/pci/virtio/virtio_gpu.c

```diff
@@ -1,6 +1,9 @@
 /*
+ * Copyright (C) OASIS Open 2018. All rights reserved.
  * Copyright (C) 2022 Intel Corporation.
  *
+ * SPDX-License-Identifier: BSD-3-Clause
```

Comment:
Is this the license that was used in the original file? If not, can we change it?

Reply:
No, this file just used some definitions about virtio-gpu spec(https://github.com/oasis-tcs/virtio-spec/blob/master/virtio-gpu.tex). We can change this file.

Reply:
> No, this file just used some definitions about virtio-gpu spec(https://github.com/oasis-tcs/virtio-spec/blob/master/virtio-gpu.tex). We can change this file.

OK, thanks for the confirmation (/cc @yaomichael )